### ISSUE TITLE
iOS real device splashscreen frozen

### ISSUE THREAD

**Issue by MumblesNZ** [ionic-team/cordova-plugin-ionic#101]
Hello,

We are trying to deploy our app and testing on iOS. When we deploy with --livereload it all works fine, but when we test in the simulator without livereload it freezes on the splashscreen. When I uninstall cordova-plugin-ionic everything seems to work fine, so I think it may be the culprit. Any clues? 

Ionic info:
```
cli packages: (/usr/local/lib/node_modules)

   @ionic/cli-utils  : 1.19.2
   ionic (Ionic CLI) : 3.20.0

global packages:

   cordova (Cordova CLI) : 8.0.0

local packages:

   @ionic/app-scripts : 3.1.11
   Cordova Platforms  : ios 4.5.4
   Ionic Framework    : ionic-angular 3.9.2

System:

   ios-sim : 7.0.0
   Node    : v8.11.3
   npm     : 5.6.0
   OS      : macOS High Sierra
   Xcode   : Xcode 9.4.1 Build version 9F2000

Environment Variables:

   ANDROID_HOME : not set

Misc:

   backend : pro
```

cordova plugin ls:
```
cordova-plugin-android-permissions 1.0.0 "Permissions"
cordova-plugin-console 1.1.0 "Console"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-email-composer 0.8.15 "EmailComposer"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-inappbrowser 3.0.0 "InAppBrowser"
cordova-plugin-ionic 5.0.0-rc.3 "cordova-plugin-ionic"
cordova-plugin-ionic-webview 2.0.0 "cordova-plugin-ionic-webview"
cordova-plugin-nativeaudio 3.0.9 "Cordova Native Audio"
cordova-plugin-speechrecognition 1.1.2 "Speech Recognition"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-tts 0.2.3 "TTS"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-sqlite-storage 2.3.3 "Cordova sqlite storage plugin"
ionic-plugin-keyboard 2.2.1 "Keyboard"
```
cordova platform ls: 
```
Installed platforms:
  ios 4.5.4
Available platforms: 
  android ~7.0.0
  browser ~5.0.1
  osx ~4.0.1
  windows ~5.0.0
  www ^3.12.0
```

**Comment by nphyatt** [comment]
Do you see any error logs in the xcode console of if you use the safari inspector?

**Comment by MumblesNZ** [comment]
Hey, thanks for the quick response. 

In safari console logs, the only error I get is: 

`
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (localhost, line 0) 
`
When it's trying to load http://localhost:8080/#/login

Also none of the resources are loaded in the resource tab. Only an empty localhost file, and a user-script:1 which has WKWebView stop scroll logic. 

As a side note, if I roll back to v3.1.3 (The version we were using before the upgrade), everything works fine.

**Comment by nphyatt** [comment]
@MumblesNZ  I think there is an issue with the simulator where if you rebuild the app without completely deleting the old version off the phone it will cause this error. I believe it has to do with us storing the absolute path to the new version rather than the relative path and when you rebuild Xcode will sometimes change the underlying path on the filesystem for the simulator. This shouldn't happen on a real device but I'm going to try to release a version today that uses relative file path and therefore should avoid this issue. 

Would you be able to try developing on a real device from Xcode and see if you're still having the issues?
Or try deleting the app completely from the simulator before rerunning?

I'll ping you when I release what I hope will be the fix.

Thanks for the report!

**Comment by nphyatt** [comment]
@MumblesNZ I released a new pre-release of the plugin that I think will fix your issue. Could you try installing `cordova-plugin-ionic@next` (`5.0.1-0`) and if you can confirm that it works I'll make it the official release?

**Comment by nphyatt** [comment]
FYI I've since published `v5.0.1` which should include the fix and is on the official `latest` tag.

**Comment by MumblesNZ** [comment]
@nphyatt, it seems to have done the trick, thanks for your help.
